fpga_interchange/arch.cc

```diff
@@ -0,0 +1,577 @@
+/*
+ *  nextpnr -- Next Generation Place and Route
+ *
+ *  Copyright (C) 2018  Clifford Wolf <clifford@symbioticeda.com>
```

Comment:
as you touch files, can you add your name to the copyright and also update claire's name?

Reply:
I believe I've fixed this, please take a look!